### DISCUSSION TITLE
Neovim and Wezterm plugin for sending config overrides from within Neovim

### DISCUSSION THREAD

**Discussion by winter-again** [wezterm/wezterm#4462] [Show and tell]
I've hacked together a simple [plugin](https://github.com/winter-again/wezterm-config.nvim) for sending (some) Wezterm config overrides from within Neovim. It also uses Wezterm's built-in plugin system for setting stuff up on that end. Like I said, it's a bit hacky and therefore I've had to make some very opinionated requirements. Check it out and let me know if you have any suggestions.

**Comment by danjessen** [comment]
Looks really great
